### PR TITLE
[KEP 3031] Add documentation about signed Kubernetes artifacts

### PR DESCRIPTION
**Pull request by saschagrunert**
Placeholder PR for the documentation of the signed Kubernetes artifacts.

Refers to https://github.com/kubernetes/website/issues/35341, https://github.com/kubernetes/enhancements/issues/3031

cc @Verolop you can takeover this PR at any time you want.

### CODE REVIEW

**Comment by krol3** [comment]
Hello @saschagrunert   👋 please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before **deadline Tuesday 15th November 2022**. Thank you!

**Comment by saschagrunert** [comment]
/unhold

Ready for review.

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:63]:
```suggestion
cosign verify registry.k8s.io/kube-apiserver-amd64:v{{< skew currentVersion >}}.0
```

Extra credit, very much not blocking any merge: look up the latest patch version too.

File: content/en/docs/concepts/security/security-checklist.md
Comment (sftim) [inline content/en/docs/concepts/security/security-checklist.md:293]: Is this change right?

**Review by sftim — COMMENTED** [review]
Here's some feedback - I hope it makes sense

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (saschagrunert) [reply content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:4]: Fixed as suggested.

File: content/en/docs/concepts/security/security-checklist.md
Comment (saschagrunert) [reply content/en/docs/concepts/security/security-checklist.md]: No, reverted it :+1:

**Review by krol3 — COMMENTED** [review]
/LGTM

**Comment by krol3** [comment]
Hi @saschagrunert ! This PR needs a doc review by **Mon Nov 28th** to get this into the release. Please reach out to required SIGs to get their review. Thank you!

**Comment by saschagrunert** [comment]
@kubernetes/release-managers @kubernetes/sig-release-leads PTAL

**Review by cpanato — APPROVED** [review]
/lgtm

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (xmudrii) [inline content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:46]: Should we use `kubectl` or `$BINARY` here, e.g:

```suggestion
cosign verify-blob "$BINARY" --signature "$BINARY.sig" --certificate "$BINARY.cert"
```

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (xmudrii) [inline content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:31]: Wasn't `COSIGN_EXPERIMENTAL` dropped just recently? Is there a new `cosign` release that doesn't require it? Also, should we mention the minimum `cosign` version in the requirements?

**Review by xmudrii — COMMENTED** [review]
Some questions, but other than that, LGTM.

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (saschagrunert) [reply content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:46]: Good catch, fixed.

File: content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md
Comment (saschagrunert) [reply content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md:31]: Yeah, I think we can just re-add it to ensure it works for each version.

**Review by xmudrii — APPROVED** [review]
/lgtm

**Review by divya-mohan0209 — COMMENTED** [review]
I've approved this from the SIG Docs side of things. @saschagrunert if you could get one of the Release folks to LGTM it once again after a re-review, we could get this merged. Thanks!
/approve

**Comment by divya-mohan0209** [comment]
Ah @xmudrii beat me to it, thanks! :)

**Pull request merged**
